templates/project/create.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base_headless.html" %}
+{% load i18n %}
+
+{% block title %}<title>Palamar | Create Project </title>{% endblock title %}
+
+{% block content %}
+
+    <div class="register-box">
+        <div class="register-logo">
+            <a href="{{ SITE_URL }}">Palamar</a>
+        </div>
+
+        <div class="register-box-body">
+            <p class="login-box-msg">Crate Your First Project</p>
+            <form action="/project/create/" method="POST">
+                <span class="text-red"> {{ form.errors.project.as_text }} </span>
+                <div class="form-group has-feedback">
+                    <input name="project" type="text" class="form-control" placeholder="Enter a project name">
+                    <span class="glyphicon glyphicon-folder-close form-control-feedback"></span>
+                </div>
+                <div class="row">
+                    <span class="text-red" style="margin-left: 15px;"> {{ form.errors.tos.as_text }} </span>
+                    <div class="col-xs-8">
+                        <div class="checkbox icheck">
+                            <label>
+                                <input name="tos" type="checkbox"> I agree to the <a href="#">terms</a>
```

Comment:
There is little confusion here. The Turkish translation is "Kullanıcı sözleşmesini kabul ediyorum." As you see, the object (kullanıcı sözleşmesi as term), is at the beginning of the sentence. How will we separate it?